src/core/dbus-execute.c

```diff
@@ -832,18 +832,37 @@ static int property_get_mount_images(
         assert(property);
         assert(reply);
 
-        r = sd_bus_message_open_container(reply, 'a', "(ssb)");
+        r = sd_bus_message_open_container(reply, 'a', "(ssba(us))");
         if (r < 0)
                 return r;
 
         for (size_t i = 0; i < c->n_mount_images; i++) {
+                MountOptions *o;
+
+                r = sd_bus_message_open_container(reply, SD_BUS_TYPE_STRUCT, "ssba(us)");
+                if (r < 0)
+                        return r;
                 r = sd_bus_message_append(
-                                reply, "(ssb)",
+                                reply, "ssb",
```

Comment:
hmm, we merged MountImages support after the last release right? if so, it's ok to change the signature here, otherwise it would be an api break...

Reply:
yes, it was merged after the release